index.html

```diff
@@ -10291,8 +10291,9 @@ <h1>Data Model</h1>
       the scope of a <a>JSON-LD document</a>. Blank node identifiers begin with
       <code>_:</code>.</li>
     <li>A <a>JSON-LD value</a> is a <a>typed value</a>, a <a>string</a> (which is interpreted
-      as <a>typed value</a> with type <code>xsd:string</code>), a <a>number</a>
+      as a <a>typed value</a> with type <code>xsd:string</code>), a <a>number</a>
       (<a>numbers</a> with a non-zero fractional part, i.e., the result of a modulo&#8209;1 operation,
+      <span class="changed">or which are too large to represent as integers,</span>
```

Comment:
I think it is important to specify what kind of integer this is (XSD, JSON, programming language integer, ...).
It looks like the XSD spec defines [the value space of `xsd:integer` as an infinite set](https://www.w3.org/TR/xmlschema-2/#integer), which means that there is no maximum (or minimum) value.
Next to that, the [JSON RFC also does not place a strict limit on integer/number sizes](https://tools.ietf.org/html/rfc7159#section-6), but merely allows limits to be set by software.

Unless this has been defined somewhere already, I think it would be good to define integer sizes in the JSON-LD spec. To be consistent with most programming languages, values for 32-bit integer sizes may be useful. Perhaps this can be a soft limit, to be in line with JSON.

Side-note: Next to mentioning values that are _too large_, values that are _too small_ should also be considered.

Reply:
Are there such limit in the RDF model? I do not remember having one, in which case I do not think that, formally, JSON-LD could impose any limit...

Reply:
Text was added to refer to Data Round Tripping for limitations on integer representations.